build-image/openhabian.conf

```diff
@@ -1,6 +1,7 @@
 # Modify your openHABian settings
 # Handle with care, only touch if you need to!!
 
+configfile=/etc/openhabian.conf
```

Comment:
I think this could lead to trouble.... Why put that in all the configs, with the risk somone things he could modify the location of the config here (which does not make sense, since the pointer to this file is anyway hardcoded).
why not
CONFIGFILE=/etc/openhabian.conf in first-boot.bash and openhabian-setup.sh? (maybe I overlooked some place where it is needed....) - the files in functions are anyway not intended to be used separately, are they?

Reply:
ok I did that and changed all references to uppercase CONFIGFILE so we do not accidentially confuse the origin of the variable